### DISCUSSION TITLE
Question about filtering advertised sessions

### DISCUSSION THREAD

**Discussion by ghost** [YimMenu/YimMenu#3329] [Q&A] [answered]
I wanted to ask a silly question. I didn't find in the file [start_matchmaking_find_sessions.cpp](https://github.com/YimMenu/YimMenu/blob/master/src/hooks/matchmaking/start_matchmaking_find_sessions.cpp) that we filter advertised sessions during matchmaking. We just add the number of instances in [view_session_browser.cpp](https://github.com/YimMenu/YimMenu/blob/master/src/views/network/view_session_browser.cpp) and do nothing more. So, does it mean we don't filter them during matchmaking, or am I wrong?

**Comment by lonelybud** [comment, marked as answer]
yes it is not filtered. Only the number of advertisements is shown in front of session info.
